gohelper.go

```diff
@@ -11,89 +11,347 @@ import (
 	"os"
 )
 
-type Relationship struct {
-	Host     string `json:"host"`
+type Error string
+
+func (e Error) Error() string { return string(e) }
+
+const notValidPlatform = Error("No valid platform found.")
```

Comment:
I believe you want `errors.New()` there?

Reply:
Do I?  This is the pattern I've seen used before.

Reply:
Specifically, I believe this is "explict cast a string to an Error, and since strings have an Error() receiver function they are now Errors, yay."

Reply:
I mean, there's specifically a function for it in the stdlib... https://golang.org/pkg/errors/#example_New